tests/Cache/CacheFileStoreTest.php

```diff
@@ -28,21 +28,6 @@ public function testNullIsReturnedIfFileDoesntExist()
         $this->assertNull($value);
     }
 
-    public function testUnserializableFileContentGetDeleted()
```

Comment:
@taylorotwell This test is does not actually represent what's happening on the FileStore class. So, it's happening was that PHPUnit 9 was overriding the exception handler, by converting notices to exceptions. Meaning that unserialize unserializable content, would result in an exception and consequentially removing the file content.

Now, with PHPUnit 10, this test would be fail, because notices would not be converted to exceptions, meaning that unserialize unserializable content would simply result in `false`.